src/DynamicMvvm.Abstractions/ViewModel/IViewModel.md

```diff
@@ -162,6 +162,20 @@ public class MyViewModel : ViewModelBase
   // command parameter when executed. The CancellationToken will be cancelled if the IViewModel is disposed.
   public IDynamicCommand MyCommandFromTaskWithParameter => this.GetCommandFromTask<int>(ExecuteMyTaskWithParameter);
   private async Task ExecuteMyTaskWithParameter(CancellationToken ct, int parameter) => Task.CompletedTask;
+  
+  // This will create and attach a new IDynamicCommand named "MyCommandForItemFromTask + GUID"
```

Comment:
Instead of repeating the same thing as before, I would just refer to `GetCommandFromTask` and explain what the _ItemCommand_ does differently.

Reply:
I think this might be the best strategy for the XML doc as well (if possible).